### PR TITLE
Feature/app structure

### PR DESCRIPTION
**Pull request by RomiKMoveo**
Created the app's structure base on the recommended structure  of Moveo-Protocol-code for React.

Added home page, and the component that the home page render( the components are not implemented yet).
Added general style, colors, and typography as a basic layer of the app.
Added un-implemented button card and error components that can be reuse throughout  the app. 

none

I have:
- [ ] deleted all irrelevant comments, logs and debuggers.
- [ ] assure that I don't print full error stacks, full body response or headers from the server in the console.
- [ ] deleted all sensitive information from the code because it's being imported from environment variables.
- [ ] encrypted keys/passwords that are sent through the internet.
- [ ] sanitized all user inputs to prevent attacks.
- [ ] made sure my code is generic and reusable for future things or reused existing resources.
- [ ] made sure that my code is readable.
- [ ] checked that the functions and resource names are self-explanatory.
- [ ] tested the code locally and it works as expected.
- [ ] added validations and error handling in relevant cases (external requests, integrations, etc).

### CODE REVIEW

File: epicure-frontend/src/pages/home/components/Hero/Hero.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import "../../../../data/types/typography.scss";
+@import "../../../../data/variables.scss";
+
+.homePage {
+    display: flex;
+    //flex-direction: column;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/Hero/Hero.scss]: Delete comments

File: epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import "../../../../data/types/typography.scss";
+@import "../../../../data/variables.scss";
+
+.homePage {
+    display: flex;
+    //flex-direction: column;
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.scss]: Delete

File: epicure-frontend/src/shared/components/Button/Button.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
+@import "../../../data/types/typography.scss", "../../../data/variables.scss", "../../../data/UI/colors.scss..";
+
+.btn {
+  //@include flex-container( row, center, center );
```
Comment (MoranOshiaMoveo) [inline epicure-frontend/src/shared/components/Button/Button.scss]: Delete

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]
Pls noticed that u'll setting node modules in to git ignore it is not supposed to be in the uploaded files

File: epicure-frontend/src/pages/home/components/Hero/Hero.scss (hunk 1 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/Hero/Hero.scss]: done

File: epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.scss (hunk 2 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/pages/home/components/IconsMeaning/IconsMeaning.scss]: delete

File: epicure-frontend/src/shared/components/Button/Button.scss (hunk 3 above)
Comment (RomiKMoveo) [reply epicure-frontend/src/shared/components/Button/Button.scss]: done

**Review by MoranOshiaMoveo — APPROVED** [review]

**Pull request merged**